package.json

```diff
@@ -40,6 +40,7 @@
   ],
   "typings": "dist/packages/index",
   "dependencies": {
+    "downshift": "3.2.10",
```

Comment:
[Downshift](https://github.com/downshift-js/downshift) is a great little library that handles keyboard navigation within the menu and other a11y goodies like updating ARIA roles